### DISCUSSION TITLE
Full page component URL changes to the wrong URL

### DISCUSSION THREAD

**Discussion by joostvanhoof** [livewire/livewire#3743] [7. Bugs (archived)] [answered]
I'm guessing that this happened after the update to `2.6.1`. This only happens on URLs that are full page Livewire components, not on regular Laravel controllers.

What happens is this:
- Navigate to `/dashboard/test`
- Then navigate to `/dashboard/another`
- The URL **in the browser's URL bar** is now `/dashboard/test` while the page **being displayed** is `/dashboard/another`

If I inspect the request with Telescope and look at the Response I can see that the `path` value is still the first page `/dashboard/test` instead of the new page where the user now is `/dashboard/another`

```json
"data": {
  "manager": {
    "response": {
      "effects": {
        "path": "https://example.test/dashboard/test"
      }
    }
  }
}
```

In the Session data the value of `url` is also incorrect, it's `/dashboard/another` while it should be `/dashboard/test`.

```json
"_previous" {
  "url": "https://example.test/dashboard/another"
}
```

Lastly, in the Header data the `referrer` has the correct value of `/dashboard/test`.

Any more info needed please let me know.

**Comment by joostvanhoof** [comment, marked as answer]
I should have read the discussion page first, this is fixed in `v2.6.2` which has just been released, thanks @joshhanley !

**Reply by joshhanley** [reply]
No worries 🙂
